src/dbus_api/blockdev/mod.rs

```diff
@@ -89,9 +102,19 @@ pub fn get_initial_properties(
             consts::BLOCKDEV_HARDWARE_INFO_PROP => shared::blockdev_hardware_info_prop(dev),
             consts::BLOCKDEV_USER_INFO_PROP => shared::blockdev_user_info_prop(dev),
             consts::BLOCKDEV_INIT_TIME_PROP => shared::blockdev_init_time_prop(dev),
-            consts::BLOCKDEV_POOL_PROP => parent,
+            consts::BLOCKDEV_POOL_PROP => parent.to_owned(),
```

Comment:
Can you give me a little more information about this change?

Reply:
Didn't compile w/out it. I didn't look closely into the macro to see why.

Reply:
Ok!

Reply:
Yeah, there's two vecs, one for each interface, each evaluated in the order that it is defined. And so ```parent``` can't be shared. I don't know that there's a prettier way to deal with this.